src/model.jl

```diff
@@ -490,48 +551,192 @@ const perihelion_parameters_phase = [251.9025, 280.8325, 128.3057, 348.1074, 292
                                      168.2438, 161.1199, 55.0196, 262.6495, 200.3284, 201.6651,
                                      294.6547, 99.8233, 213.5577, 154.1631, 232.7153, 138.3034,
                                      204.6609, 106.5938, 250.4676, 332.3345, 27.3039]
-
-
-
-# Eqn. 4 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function e_sincos_pi(year)
+  
+  
+  
+  # Eqn. 4 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function e_sincos_pi(year)
   M    = eccentricity_parameters_amplitude
   g    = eccentricity_parameters_mean_rate
   beta = eccentricity_parameters_phase
-
+  
   arg = deg2rad.(g/3600) * (year-1950) .+ deg2rad.(beta)
-
+  
   return sum(M .* sin.(arg)), sum(M .* cos.(arg))
-end
-
-eccentricity(year, e_sin_pi, e_cos_pi) = sqrt(e_sin_pi^2 + e_cos_pi^2)
-
-eccentricity(year) = eccentricity(year, e_sincos_pi(year)...)
-
-
-# Eqn. 1 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function obliquity(year)
+  end
+  
+  eccentricity(year, e_sin_pi, e_cos_pi) = sqrt(e_sin_pi^2 + e_cos_pi^2)
+  
+  eccentricity(year) = eccentricity(year, e_sincos_pi(year)...)
+  
+  
+  # Eqn. 1 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function obliquity(year)
   A     = obliquity_parameters_amplitude
   f     = obliquity_parameters_mean_rate
   delta = obliquity_parameters_phase
-
+  
   epsilon_star = 23.320556
-
+  
   return deg2rad(epsilon_star + sum(A/3600 .* cos.(deg2rad.(f/3600) * (year-1950) .+ deg2rad.(delta))))
-end
-
-
-# Eqn. 6 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
-function perihelion(year)
+  end
+  
+  
+  # Eqn. 6 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+  function perihelion(year)
   F     = perihelion_parameters_amplitude
   f     = perihelion_parameters_mean_rate
   delta = perihelion_parameters_phase
-
+  
   pi_ = atan(e_sincos_pi(year)...)
   psi_tilde = 50.439273
   zeta = 3.392506
-
+  
   psi = psi_tilde/3600 * (year-1950) + zeta + sum(F/3600 .* sin.(deg2rad.(f/3600) * (year-1950) .+ deg2rad.(delta)))
-
+  
   return (pi_ + deg2rad(psi)) % (2*pi)
+  end
+
+
+"""
+read_sea_ice_extent_N()
+* Default year: 1979
+* Possible years to pick annual sea ice extent for the Northern Hemisphere : 1979 to 2020
+* returns a vector with sea ice extent per month from March to February
+
+"""
+
```

Comment:
```suggestion
```
For the docstrings to work, they have to be just above the function they document. For example:
```
julia> compute_equilibrium!(
compute_equilibrium!(discretization; max_years, rel_error, verbose, update_heat_capacity, update_solar_forcing) in Klimakoffer at /mnt/hd1/scratch/aruedara/20220201_Klimakoffer_HeatCapacity/Klimakoffer.jl_xenia/src/numerics.jl:24
help?> compute_equilibrium!
search: compute_equilibrium!

  compute_equilibrium!(...)

    •  rel_error is the tolerance for global temperature equilibrium (default is 2e-5).

    •  max_years is the maximum number of annual cycles to be computed when searching for equilibrium

    •  updateheatcapacity : manage the monthly update function for the heat capacity

    •  updatesolarforcing : manage the monthly update function for the solar forcing

```